test/UnitTest/Microsoft.AspNet.OData.Test/PublicApi/Microsoft.AspNet.OData.PublicApi.bsl

```diff
@@ -104,6 +104,7 @@ public abstract class Microsoft.AspNet.OData.EdmStructuredObject : Delta, IDynam
 	public virtual Microsoft.OData.Edm.IEdmTypeReference GetEdmType ()
 	public virtual System.Collections.Generic.IEnumerable`1[[System.String]] GetUnchangedPropertyNames ()
 	public System.Collections.Generic.Dictionary`2[[System.String],[System.Object]] TryGetDynamicProperties ()
+	public System.Collections.Generic.Dictionary`2[[System.String],[System.Object]] TryGetInstanceAnnotationDictionary ()
 	public virtual bool TryGetPropertyType (string name, out System.Type& type)
```

Comment:
Where does this come from? I don't see any change to EdmStructuredObject.cs.

Reply:
#Resolved -removed